main.py

```diff
@@ -1,10 +1,26 @@
 """Arquivo principal que será interpretado pelo interpretador."""
-
+import math
 
 def main():
     """Função principal que será rodada quando o script for passado para o interpretador."""
-    # COLOQUE SEU CÓDIGO AQUI
-    # TODO
+    Ano = int(input("Digite o ano desejado\n"))
+    k = 0
+    if Ano % 4 == 0:
+      k = 1
+    
+
+    if Ano % 100 == 0:
+      k = 0
+
+
+    if Ano % 400 == 0:
+      k = 1
+
+
+    if k == 1:
```

Comment:
Como você está usando uma variável que essencialmente é booleanda poderia trocar para ficar mais legível:
```suggestion
    Ano = int(input("Digite o ano desejado\n"))
    k = False
    if Ano % 4 == 0:
      k = True
    

    if Ano % 100 == 0:
      k = False


    if Ano % 400 == 0:
      k = True


    if k:
```
Vou dar um merge, mas fica a dica para as próximas.